docs/administration-guide/databases/mongodb.md

```diff
@@ -0,0 +1,50 @@
+# Working with MongoDB in Metabase
+
+This document is relevant for Metabase versions equal to or newer than `0.32.9`.
+
+We have received many questions and support requests to assist with configuration on connecting to a MongoDB instance/cluster.
+
+In this guide, we document:
+
+ - General connectivity concerns.
+ - Connecting to a MongoDB Atlas cluster.
+
+ ## General connectivity concerns
+ 
+ - Have you checked your cluster host whitelist?  When testing a connection but seeing failure, have you tried setting the IP whitelist to `0.0.0.0/0`?  This allows connections from any IP addresses. If you know the IP address(es) or CIDR block of clients, use that instead.
+ 
+ - Are you using self-signed certificates?  While we don't support providing one through the Metabase UI, we can use the command line to configure:
+    Copy existing store and add self-signed cert to it:
+     
+    ```bash
+    cp /usr/lib/jvm/default-jvm/jre/lib/security/cacerts ./cacerts.jks
+    keytool -import -alias cacert -storepass changeit -keystore cacerts.jks -file my-cert.pem
+    ```
+    
+     Then, start Metabase using the store:
+     
+    ```bash
+    java -Djavax.net.ssl.trustStore=cacerts.jks -Djavax.net.ssl.trustStorePassword=changeit -jar metabase.jar
+    ```
+ 
+     Additional reference for configuring SSL with MongoDB for self-signed certs:
+      - http://mongodb.github.io/mongo-java-driver/3.0/driver/reference/connecting/ssl/
+ 
+ ## Connecting to a MongoDB Atlas cluster
+ 
+ To make sure you are using the correct connection configuration:
+  1. Log into your [Atlas cluster](https://cloud.mongodb.com)
+  
+  2. Select the cluster you want to connect to, and click "Connect"
+
+     ![Your cluster screengrab](mongo_1.jpg "Your cluster")
+
+  3. Click "Connect Your Application"
+
+     ![Connect screengrab](mongo_2.jpg "Connect")
+
+  4. Select "Java" and "3.6 or later"
+
+     ![Java screengrab](mongo_3.jpg "Java")
+
+  5. The resulting connection string has the relevant information to provide to the Metabase add a database for MongoDB admin page.  You will likely want to select the Metabase option `Use DNS SRV`, which Atlas clusters use by default.
```

Comment:
> The resulting connection string has the relevant information to provide to the Metabase add a database for MongoDB admin page.

Would prefer "The resulting connection string has the relevant information to add to Metabase's `Add a Database` form for MongoDB."